src/test/java/org/apache/commons/collections4/bidimap/AbstractBidiMapTest.java

```diff
@@ -265,16 +266,18 @@ private void removeValue(final BidiMap<?, ?> map, final Object value) {
 
     @Test
     public void testBidiKeySetValuesOrder() {
-        resetFull();
-        final Iterator<K> keys = map.keySet().iterator();
-        final Iterator<V> values = map.values().iterator();
-        while (keys.hasNext() && values.hasNext()) {
-            final K key = keys.next();
-            final V value = values.next();
-            assertSame(map.get(key), value);
-        }
-        assertFalse(keys.hasNext());
-        assertFalse(values.hasNext());
+        if((getIterationBehaviour() & AbstractCollectionTest.UNORDERED) == 0){
```

Comment:
Using an if statement does not allow reporting that the test has been skipped. Can you revert and use the previous suggestion to skip based on an assumption (Junit5):
```Java
public void testBidiKeySetValuesOrder() {
    // Skip if collection is unordered
    Assumptions.assumeFalse((getIterationBehaviour() & AbstractCollectionTest.UNORDERED) != 0);
```

Reply:
@aherbert Got it. Updated